app/assets/javascripts/pretty-text/addon/sanitizer.js

```diff
@@ -143,6 +143,15 @@ export function sanitize(text, allowLister) {
         }
       }
     },
+    onTag(tag, html) {
+      if (
+        tag === "video" &&
+        html.match(/ autoplay/i) &&
+        !html.match(/ muted/i)
+      ) {
+        return "[removed]";
```

Comment:
Can you automatically add the muted attribute instead?

Reply:
That is possible and I guess it's better?